toolvoice/scripts/inference/qwen_omni_infer.py

```
@@ -0,0 +1,190 @@
+        audios, _, _ = process_mm_info(messages, use_audio_in_video=False)
+    except Exception:
+        return None
+    inputs = {"prompt": text, "multi_modal_data": {"audio": audios if audios else []}}
+    if model_version != "2.5":
+        inputs["mm_processor_kwargs"] = {"use_audio_in_video": False}
+
+    outputs = llm.generate([inputs], sampling_params=sampling_params)
+    return outputs[0].outputs[0].text
+
+
+def main():
```

Ruff (0.16.1)

[warning] Too many branches (15 > 12)

(PLR0912)